### ISSUE TITLE
ETSY004 does not report multiple violations if one is ok

### ISSUE THREAD

**Issue by pmacdougall** [etsy/foodcritic-rules#4]
If you have a recipe with multiple ETSY004 violations, then correct one of them by adding a conditional, the rest of the violations will no longer be reported.

``` ruby
execute "fetch_file" do
  command "wget -O /tmp/foo.tar.gz http://me.mycorp.com/files/foo.tar.gz"
end

execute "fetch_file" do
  command "wget -O /tmp/foo.tar.gz http://me.mycorp.com/files/foo.tar.gz"
end
```

results in 

```
$ foodcritic -t ETSY004 -I ~/src/etsy/foodcritic-rules/rules.rb .
ETSY004: Execute resource defined without conditional or action :nothing: ./recipes/default.rb:1
ETSY004: Execute resource defined without conditional or action :nothing: ./recipes/default.rb:5
$
```

Changing the recipe to

``` ruby
execute "fetch_file" do
  command "wget -O /tmp/foo.tar.gz http://me.mycorp.com/files/foo.tar.gz"
  not_if { File.exists?("/tmp/foo.tar.gz") }
end

execute "fetch_file" do
  command "wget -O /tmp/foo.tar.gz http://me.mycorp.com/files/foo.tar.gz"
end
```

results in

```
$ foodcritic -t ETSY004 -I ~/src/etsy/foodcritic-rules/rules.rb .

$
```

FYI

```
$ gem list | grep foodcritic
foodcritic (1.6.1, 1.4.0, 1.3.0)
```
